libs/sdk-core/src/models.rs

```diff
@@ -159,6 +173,18 @@ impl Config {
             default_lsp_id: Some(String::from("ea51d025-042d-456c-8325-63e430797481")),
             api_key: None,
             maxfee_percent: 0.5,
+            moon_pay_config: MoonPayConfig {
```

Comment:
I prefer to not expose the config, we actually don't need a config now as we use all the values hard coded.